package.json

```diff
@@ -38,11 +38,13 @@
     "test": "jest"
   },
   "devDependencies": {
-    "babel-cli": "^6.24.1",
-    "babel-core": "^6.25.0",
-    "babel-eslint": "^7.2.3",
-    "babel-plugin-syntax-jsx": "^6.18.0",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.24.1",
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-syntax-jsx": "^7.0.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.0.0",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-eslint": "^9.0.0",
+    "babel-jest": "^23.4.2",
```

Comment:
Also, let's remove this from here. It should be included with `jest`.

Reply:
it is not, this is why tests were failing (https://github.com/avaly/babel-plugin-inline-classnames/pull/3#discussion_r212820321)

Reply:
https://jestjs.io/docs/en/getting-started#using-babel

Reply:
The following changes work fine for me on your branch: https://gist.github.com/avaly/f5760142157d618dfafcc2d27cce56ff

Reply:
Ohh, my mistake. I had the files cached by jest. Indeed, it does not run without those packages.

But let's at least change the `babel-core` dependency to `babel-core@^7.0.0-0` according to the jest docs.